crossopt/assignment1/Q2.py

```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env python3
 
 
-class Node:
+class Node(object):
```

Comment:
The "(object)" is necessary in Python2, but if you write Python 3, then it is not needed.